client/my-sites/customer-home/cards/tasks/site-setup-list/index.jsx

```diff
@@ -59,6 +59,10 @@ const startTask = ( dispatch, task, siteId ) => {
 	if ( task.actionDispatch ) {
 		dispatch( task.actionDispatch( ...task.actionDispatchArgs ) );
 	}
+
+	if ( task.actionAdvanceToNext ) {
+		advanceToNextInCompleteTask();
```

Comment:
Instead of passing a function we could pass the `localDispatch` from `<SiteSetupList>` and trigger the requisite action. 

Alternatively we could move all the selection state into Redux but that would be a larger PR and task.